pkg/executors/docker_compose_executor.go

```diff
@@ -453,6 +454,7 @@ func (e *DockerComposeExecutor) pullDockerImages() int {
 	directive := "Pulling docker images..."
 	commandStartedAt := int(time.Now().Unix())
 
+	e.SubmitStats("semphoreci/android", "compose.docker.pull.rate", []string{"semaphoreci/android"}, 1)
```

Comment:
Please refactor this call and encapsulate these details in a function as you did with Docker Pull Time.